Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteProjectController.java

```diff
@@ -215,5 +224,104 @@ public String visualizeAllNetworks(@PathVariable("projectUnixName") String proje
+	@RequestMapping(value = "sites/{projectUnixName}/statistics", method = RequestMethod.GET)
+	public String showProjectStatistics(@PathVariable("projectUnixName") String projectUnixName,
+									   Model model)throws JAXBException, QuadrigaStorageException {
+		IProject project = getProjectDetails(projectUnixName);
+
+		if (project == null) {
+			return "auth/accessissue";
+		}
+		
+		
+        String projectid = project.getProjectId();
+        List<INetwork> Networks = networkmanager.getNetworksInProject(projectid);
+        HashMap<String,Integer> top10Concepts = new HashMap<String, Integer>();
+        HashMap<String,String[]> top10ConceptsDetails = new HashMap<String, String[]>();
+        JSONArray jo = new JSONArray();
+        StringBuffer errorMsg = new StringBuffer();
+        StringBuffer successMsg = new StringBuffer();
+        //List<String> networkNames = null;
+        if(!Networks.isEmpty()){
+            
+            for (int i = 0; i < Networks.size(); i++) {
+                Network n  = (Network) Networks.get(i);
+                ITransformedNetwork transformedNetwork = transformationManager.getTransformedNetwork(n.getNetworkId());
+                String json = null;
+                if(transformedNetwork!=null){
+                    List<Link> links = transformedNetwork.getLinks();
+                    for(int itr=0; itr< links.size(); itr++){
+                        Node s = links.get(itr).getSubject();
+                        String url = s.getConceptId();
+                        String des = s.getDescription();
+                        String lbl = s.getLabel(); 
+                        if(top10Concepts.containsKey(url)){
+                            top10Concepts.put(url, top10Concepts.get(url)+1);
+                        }else{
+                            top10Concepts.put(url, 1);
+                            top10ConceptsDetails.put(url, new String[] {des, lbl});
+                        }
+                    }
+                    
+                }
+             
+            }
+            try {
+                jo = sortByValue(top10Concepts, top10ConceptsDetails);
+                model.addAttribute("show_success_alert", true);
+                successMsg.append("Top10 Concepts from network fetched successfully");
+                successMsg.append("\n");
+                System.out.print(jo.toString());
+                model.addAttribute("jsonstring", jo);
+                model.addAttribute("networkid", "\"\"");
+                model.addAttribute("project", project);
 
+            }catch(JSONException e){
+                model.addAttribute("show_error_alert", true);
+                errorMsg.append(e.getMessage());
+                errorMsg.append("\n");
+                
+            }
+        }
+
+        model.addAttribute("success_alert_msg", successMsg.toString());
+        model.addAttribute("error_alert_msg", errorMsg.toString());
+        
+		return "sites/project/statistics";
+	}

```

Comment:
I think it's when you mix tabs and spaces when Github gets confused. Yes, 4 spaces should do it.